src/components/Badge/Badge.js

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import styles from './style.scss';
+
+function Badge(props) {
+  const badgeType = `badge--${props.type || 'primary'}`;
```

Comment:
Other components provides `className` prop, I'm not sure if badge should provide it to. cc: @kamilmateusiak

Reply:
yup, we are trying to make our component as flexible as possible. Usually, we are using {...restProps}. That way developer will be able to pass e.g `data-test` attribute. Also, we add a possibility to extend/change our styles by merging our className with incoming. You can use `classnames` library for this.